src/texture/tga.rs

```diff
@@ -0,0 +1,76 @@
+use super::Texture;
```

Comment:
Why are you using unsafe code here?

You can perform the same kind of operation with slices.

As a note: you should consider using clippy [undocumented-unsafe-blocks](https://rust-lang.github.io/rust-clippy/master/#undocumented_unsafe_blocks) linter and document behaviors of those.

Reply:
I think that's the wrong line for your comment?

Reply:
That comment is for the whole file.

Reply:
Ah ok

Reply:
> As a note: you should consider using clippy [undocumented-unsafe-blocks](https://rust-lang.github.io/rust-clippy/master/#undocumented_unsafe_blocks) linter and document behaviors of those.

I do remember getting some Clippy lints about undocumented unsafe code in some Rust projects. If it's not there by default I should consider crafting up a stricter linting configuration (maybe stricter formatting too).